tck/src/main/java/org/eclipse/microprofile/lra/tck/participant/api/LRATypeTckConfig.java

```diff
@@ -0,0 +1,42 @@
+/*
+ *******************************************************************************
+ * Copyright (c) 2020 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * You may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *******************************************************************************/
+package org.eclipse.microprofile.lra.tck.participant.api;
+
+public class LRATypeTckConfig {
```

Comment:
This class makes it difficult to navigate from the test to the resource that the test is using. Before I could be looking at the test in the IDE and click the resource path which takes me to the resource. Adding this level of redirection now means I have to work a lot harder to find the resource. Since the change is not tied to the issue can we go back to the way it was before.

Reply:
I guess it is because more than one resource now has this path directly? Perhaps this new file could be used to provide a consistently named list of constants and each of the different resources reference it where necessary. e.g. in LraTypeResource:
```public static final String REQUIRED_PATH = LRATypeTckConfig.REQUIRED_PATH;```

Reply:
The constants are specific to the resource where they are used (if they happen to be the same string then that is coincidental). Putting them in a separate class just obfuscates the tests.

Reply:
so then TckLRATypeTests class will have 3 imports for each of the path parameters (as Tom mentioned same strings) just to be able to move to the right method from the test in one click? Isn't it a little redundant? I can, for instance, add a comment with:

```
    /**
     * {@link LRATypeTckResource#requiredLRA}
     */
```
to each test which can avoid duplication. Thougts?

Reply:
What is wrong with separate imports. The problem with your approach is as I stated above, the path belongs with the resource where it is used. Putting it somewhere else is counter intuitive and makes navigation awkward.

Reply:
I think one thing to clarify is whether these strings definitely have the exact same purpose. Is it possible two different services might need to vary any of them in the future. e.g "/required" and "/required2" on two different services?

Reply:
Actually if they were different services they would likely have a different root `@Path` to separate them. So it would be whether you might want one service to have two REQUIRED_PATH values.

Reply:
They have different root paths so not really. I am already in the process of changing this through. I will push the upgrade later today.